neurom/core/morphology.py

```diff
@@ -511,10 +489,6 @@ def iter_sections(self, order=Section.ipreorder, neurite_order=NeuriteIter.FileO
         """
         return iter_sections(self, iterator_type=order, neurite_order=neurite_order)
 
-    def __nonzero__(self):
-        """If has root node."""
-        return bool(self.morphio_root_node)
-
```

Comment:
@mgeplf you mentioned you don't agree with removing the section booleaness. Is this still true?

Reply:
no; I can see how it's confusing; one can check if there are children more explicitly.